loop_rate: 100.0
urdf_spawn_pose: [0.0, 0.0, 0.0] # x,y,z
tracking_cam: False 
cam_angle: [2.0, 0, -45] # dist, yaw, pitch
max_torque: 16 # Nm, changed for R-snake - need to make this parameter as specific to snake
joint_control_mode: 0 # 0: position, 1: velocity, 2: torque control
#terrain_path: "path_to_your_urdf"
